features/borrowers_view_and_sign_deed.feature

```diff
@@ -0,0 +1,280 @@
+(US133) Confirmation Message displayed to borrower on a subsequent page.
+(US133) Each borrower must be able to view a deed multiple times but must only be able to confirm the details once.
+(US133) When the deed is viewed subsequently it will show that the relevant borrower has confirmed the deed.
+
+@us133
+Scenario: Single borrower views and signs deed, then views again
+  Given I have valid deed data with <1> borrowers
+  And I create the deed via the Deed API
+  And borrower <1> views the deed
+  And the borrower signature element <1> is present on page
+  Then the deed is digitally signed by borrower <1>
+  And borrower <1> views the deed again
+  Then I verify borrower <1> has signed the deed
+
+@us133
+Scenario: Single borrower views deed, then views and signs
+  Given I have valid deed data with <1> borrowers
+  And I create the deed via the Deed API
+  And borrower <1> views the deed
+  And borrower <1> views the deed again
+  Then the deed is digitally signed by borrower <1>
+
+@us133
+Scenario: Single borrower views deed 3 times
+  Given I have valid deed data with <1> borrowers
+  And I create the deed via the Deed API
+  And borrower <1> views the deed
+  And borrower <1> views the deed again
+  Then borrower <1> views the deed again
+
+@us133
+Scenario: 2 Borrowers can view and sign a deed. p1
+  Given I have valid deed data with <2> borrowers
+  And I create the deed via the Deed API
+  And borrower <1> views the deed
+  Then the deed is digitally signed by borrower <1>
+  And borrower <2> views the deed again
+  Then I verify borrower <1> has signed the deed
+  Then the deed is digitally signed by borrower <2>
+
+@us133
+Scenario: 2 Borrowers can view and sign a deed. p2
+  Given I have valid deed data with <2> borrowers
+  And I create the deed via the Deed API
+  And borrower <1> views the deed
+  And borrower <2> views the deed again
+  Then the deed is digitally signed by borrower <2>
+  And borrower <1> views the deed again
+  Then I verify borrower <2> has signed the deed
+  Then the deed is digitally signed by borrower <1>
+
+@us133
+Scenario: 2 Borrowers can view and sign a deed. p3
+  Given I have valid deed data with <2> borrowers
+  And I create the deed via the Deed API
+  And borrower <2> views the deed
+  And borrower <1> views the deed again
+  Then the deed is digitally signed by borrower <1>
+  And borrower <2> views the deed again
+  Then I verify borrower <1> has signed the deed
+  Then the deed is digitally signed by borrower <2>
+
+@us133
+Scenario: 2 Borrowers can view and sign a deed. p4
+  Given I have valid deed data with <2> borrowers
+  And I create the deed via the Deed API
+  And borrower <1> views the deed
+  Then the deed is digitally signed by borrower <1>
+  And borrower <2> views the deed again
+  Then I verify borrower <1> has signed the deed
+  Then the mortgage deed is displayed
+  And borrower <2> views the deed again
```

Comment:
I'm not sure if this step needs to be here either as surely the borrower two is already on the view deed page from the previous steps and so can just go ahead and sign.